### ISSUE TITLE
Revamp the README.md

### ISSUE THREAD

**Issue by wijnsema** [Breeze/breeze-client#25]
Being a long time and every day user of the breeze.js library I have been following the progress of the new Typescript based 2.0 version, now called breeze-client for some strange reason.

Much by accident and to my surprise I stumbled the on  [Breeze/northwind-core-ng-demo](https://github.com/Breeze/northwind-core-ng-demo) which is full of beautiful demo's and examples! Yes, breeze is very much alive!

But not even the slightest mention off these examples on the main breeze client repo, this one. In stead a lot of useless information (who would ever want to build docs when they're online????) about edge cases.

My suggestion for the README.md: Just a description of what this library is about and how to install it. And just working links to the docs and samples. The rest will only add to the chaos of tow versions breeze at the moment (no criticism).

I think this library is great and deserves a better front page!

Kind regards,
Ronald Wijnsema

**Comment by steveschmitt** [comment]
Hi Ronald,
You are right: our README needs work.  We created it when we started the project, as notes to ourselves.  We forgot that it's the front page of the repo and, for a lot of users, the first impression.  People generally want to know how and why to _use_ Breeze, not how to build it themselves.

About the name: after some debate, we named the 2.0 repo "breeze-client" to match the name of the [npm package](https://www.npmjs.com/package/breeze-client) and to distinguish it from the various server-side libraries (for .NET, node, etc.) that work with the client.

We will work on improving the README with the changes you suggest.  Thanks for taking the time to comment, and thanks for using Breeze.
Sincerely,
Steve Schmitt
